pkg/operator/ceph/csi/spec.go

```diff
@@ -311,5 +316,49 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 			return fmt.Errorf("failed to create rbd service: %+v\n%+v", err, cephfsService)
 		}
 	}
+
+	if ver.Major > KubeMinMajor || ver.Minor >= provDeploymentSuppVersion {
```

Comment:
I think you'll need a version check like this:
```
if ver.Major > KubeMinMajor || (ver.Major == KubeMinMajor && ver.Minor >= provDeploymentSuppVersion {
```

Reply:
done